api/jwt/jwt.go

```diff
@@ -131,10 +131,11 @@ func (service *Service) ParseAndVerifyToken(token string) (*portainer.TokenData,
 				Role:                portainer.UserRole(cl.Role),
 				Token:               token,
 				ForceChangePassword: cl.ForceChangePassword,
-			}, nil
+			}, cl.Id, time.Unix(cl.ExpiresAt, 0), nil
 		}
 	}
-	return nil, errInvalidJWTToken
+
+	return nil, "", time.Time{}, errInvalidJWTToken
```

Comment:
In the PR against the develop branch, it will provide a `TokenData` here. Is it intentional not to include it here?

Reply:
Yes, it is intentional, this branch does not have the clean-up commit, to reduce risk. The `TokenData` is still returned here: https://github.com/portainer/portainer/pull/12165/files#diff-92c1c3e3a0962ca6ea40089005d63612b3d06c0c2aea72a3512e3f518bf2dfffR128.

Both PRs are covered by unit tests.